azkaban-common/src/main/java/azkaban/imagemgmt/models/ImageVersion.java

```diff
@@ -165,13 +168,31 @@ public static State fromStateValue(final String stateValue) {
             .filter(state -> (state.equals(State.NEW) || state.equals(State.ACTIVE)))
             .collect(Collectors.toSet());
 
+    /**
+     * Create set of new, active and test state
+     */
+    private static final Set<State> newActiveAndTestState =
+        Arrays.stream(State.values())
+            .filter(state -> (state.equals(State.NEW) || state.equals(State.ACTIVE)
+                || state.equals(State.TEST)))
```

Comment:
Are we inserting new azkaban-base version from CICD so that it can be tested?

Reply:
Yes we are inserting azkaban-base image test version from the Azkaban CICD orca job.